cfme/tests/infrastructure/test_vm_power_control.py

```diff
@@ -142,58 +172,7 @@ def _wait_for_timestamp_refresh():
         except TimedOutError:
             return False
 
-    def _check_power_options_when_on(self, soft_assert, vm, bug, from_details):
-        soft_assert(
-            vm.is_pwr_option_available_in_cfme(option=vm.POWER_OFF, from_details=from_details))
-        soft_assert(
-            not vm.is_pwr_option_available_in_cfme(
-                option=vm.POWER_ON, from_details=from_details))
-        soft_assert(
-            vm.is_pwr_option_available_in_cfme(option=vm.SUSPEND, from_details=from_details))
-
-        # limited options for SCVMM providers
-        if vm.provider.type != "scvmm":
-
-            # RHEV VMs are slightly different in terms of options
-            if vm.provider.type == "rhevm":
-                soft_assert(
-                    not vm.is_pwr_option_available_in_cfme(
-                        option=vm.GUEST_RESTART, from_details=from_details))
-                soft_assert(
-                    not vm.is_pwr_option_available_in_cfme(
-                        option=vm.RESET, from_details=from_details))
-            else:
-                soft_assert(
-                    vm.is_pwr_option_available_in_cfme(
```

Comment:
vm.is_power_option_available_in_cfme - loads details for each option and check if option isn't greyed.not the best way for us - as some options should be greyed